csm_web/scheduler/views.py

```diff
@@ -298,3 +302,30 @@ class UserViewSet(*viewset_with('list')):
 
     def list(self, request):
         return Response(self.queryset.values_list('email', flat=True))
+
+class MentorBioInfoDetail(APIView):
```

Comment:
There's probably an easier way to do this with ViewSet for consistency with the rest of the views